src/cmdInventoryExport.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type HostVars map[string]interface{}
+type Hosts map[string]HostVars
+
+type Group struct {
+	Hosts []string               `json:"hosts,omitempty"`
+	Vars  map[string]interface{} `json:"vars,omitempty"`
+}
+
+type Groups map[string]Group
+
+type Meta struct {
+	Hostvars Hosts `json:"hostvars"`
+}
+
+type AnsibleInventory struct {
+	Groups Groups `json:"groups"`
+	Meta   Meta   `json:"_meta"`
+}
+
+func (a AnsibleInventory) MarshalJSON() ([]byte, error) {
+	aux := make(map[string]interface{})
+
+	for key, value := range a.Groups {
+		aux[key] = value
+	}
+
+	aux["_meta"] = a.Meta
+
+	return json.Marshal(aux)
+}
+
+type inventoryAnsibleCmd struct {
+	List bool `long:"list" description:"List Ansible Inventory"`
```

Comment:
Not sure what that switch does, need better description

Reply:
The list flag is passed in by ansible when it runs the dynamic inventory, but because of how this is being called I think that I can just hack it to always display the inventory.

I'll drop the flag.